modules/mgmt-about-node-maintenance-schedules.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/mgmt-node-maintenance.adoc
+
+[id="mgmt-about-node-maintenance-schedules_{context}"]
+= About node maintenance schedules for bare metal nodes
```

Comment:
I'm not a native English speaker, but I think "schedules" is not correct for this. Scheduling could be understood as "I can plan some maintenance in future", not? And that's not possible with NMO. You can only start maintenance "right now".

Reply:
Yes makes sense. Will fix.